### PR TITLE
fix(plasma-ui): Omit types for HeaderLogo

### PR DESCRIPTION
**Pull request by Yakutoc**
## Проблема

В React 18 и начиная с @types/react@18.0.25 появились типы:

**onResize**: An [Event handler](https://react.dev/reference/react-dom/components/common#event-handler) function. Fires when video changes size.

**onResizeCapture**: A version of onResize that fires in the [capture phase.](https://react.dev/learn/responding-to-events#capture-phase-events)

**nonce**:  [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce)

Для пользователей оставшихся на, как пример: 

```json
{
"@types/react": "16.9.38",
"react": "17.0.2",
"react-dom": "17.0.2"
}
``` 

ломается компилция TS 

Снимок экрана 2023-03-22 в 12 38 19

## Решение  

Явно сузить тип для `HeaderLogoProps` и исключить ` 'onResize' | 'onResizeCapture' | 'nonce'`. 

Так как под капотом TS через Pick выбирает все свойства и атрибуты у `Pick<import("@salutejs/plasma-core")
    .ImageProps, "base" | "slot" | "style" | "title" | "className" | "color" | "id" ....`

-------

Примечание: 

[[Popper] Incompatible props: onResize and onResizeCapture](https://github.com/mui/material-ui/issues/35287) -здесь сталкиваются с такой же проблемой, но предлагают другие решение. 

- Для тех кто использует React 18, обновить @types/react до 18.0.25
- Для тех кто использует React 17, явно указывать returnType для проблемных мест    

----------  

:baby_chick: Download canary assets:

[colors--canary.434.4487598826.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.434.4487598826.xml)  
[color_metro_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.434.4487598826.swift)  
[color_metro_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.434.4487598826.kt)  
[color_metro_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.434.4487598826.ts)  
[color_sberHealth_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.434.4487598826.swift)  
[color_sberHealth_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.434.4487598826.kt)  
[color_sberHealth_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.434.4487598826.ts)  
[color_sbermarket_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.434.4487598826.swift)  
[color_sbermarket_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.434.4487598826.kt)  
[color_sbermarket_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.434.4487598826.ts)  
[color_sberprime_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.434.4487598826.swift)  
[color_sberprime_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.434.4487598826.kt)  
[color_sberprime_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.434.4487598826.ts)  
[color_selgros_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.434.4487598826.swift)  
[color_selgros_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.434.4487598826.kt)  
[color_selgros_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.434.4487598826.ts)  
[color_smbusiness_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.434.4487598826.swift)  
[color_smbusiness_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.434.4487598826.kt)  
[color_smbusiness_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.434.4487598826.ts)  
[PlasmaTokensColor--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.434.4487598826.swift)  
[shadow_sbermarket_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.434.4487598826.ts)  
[typo_mage_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.434.4487598826.swift)  
[typo_mage_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.434.4487598826.kt)  
[typo_mage_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.434.4487598826.ts)  
[typo_plasma_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.434.4487598826.swift)  
[typo_plasma_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.434.4487598826.kt)  
[typo_plasma_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.434.4487598826.ts)  
[typo_ruler_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.434.4487598826.swift)  
[typo_ruler_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.434.4487598826.kt)  
[typo_ruler_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.434.4487598826.ts)  
[typo_sage_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.434.4487598826.swift)  
[typo_sage_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.434.4487598826.kt)  
[typo_sage_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.434.4487598826.ts)  
[typo_sbermarket_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.434.4487598826.swift)  
[typo_sbermarket_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.434.4487598826.kt)  
[typo_sbermarket_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.434.4487598826.ts)  
[typo_soulmate_ios-swift--canary.434.4487598826.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.434.4487598826.swift)  
[typo_soulmate_kotlin--canary.434.4487598826.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.434.4487598826.kt)  
[typo_soulmate_react-native--canary.434.4487598826.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.434.4487598826.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.145.1-canary.434.4487598826.0
  npm install @salutejs/plasma-ui@1.178.1-canary.434.4487598826.0
  # or 
  yarn add @salutejs/plasma-temple@1.145.1-canary.434.4487598826.0
  yarn add @salutejs/plasma-ui@1.178.1-canary.434.4487598826.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-434/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-434/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-434/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-434/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-434/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-434/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-434/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-434/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-434/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-ui/src/components/Header/HeaderLogo.tsx [hunk 1]
```diff
@@ -2,7 +2,10 @@ import styled from 'styled-components';
 
 import { Image, ImageProps } from '../Image';
 
-export type HeaderLogoProps = Omit<ImageProps, 'height' | 'ratio' | 'customRatio'>;
+export type HeaderLogoProps = Omit<
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Header/HeaderLogo.tsx:5]: /** INFO: because react/types bla-bla */

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-434/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-434/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-434/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-434/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-434/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-434/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-434/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**